.idea/pysd.iml

```diff
@@ -2,8 +2,10 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="jdk" jdkName="Python 3.6.0 (~/anaconda/bin/python)" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="Python 3.6.0 (C:\Users\jmalar1\AppData\Local\Programs\Python\Python36-32\python.exe)" jdkType="Python SDK" />
```

Comment:
It will be good to revert changes in local work space files, like this.

Reply:
I tried excluding the .idea folder from my push entirely (consider removing from version control?) Unfortunately, I seem to have failed. If you can help please do!